README.md

```diff
@@ -32,6 +32,19 @@ Start with [**'Hello, Client!'**](https://developers.braintreepayments.com/start
 
 Next, read the [**full documentation**](https://developers.braintreepayments.com/guides/overview) for information about integration options, such as Drop-In UI, PayPal and credit card tokenization.
 
+## Versions
+
+This SDK abides by our Client SDK Deprecation Policy. For more information on the potential statuses of an SDK check our [developer docs](http://developers.braintreepayments.com/guides/client-sdk/deprecation-policy).
+
+<!-- TODO: Update TBA dates when v4 GA releases. -->
+| Major version number | Status | Released | Deprecated | Unsupported |
+| -------------------- | ------ | -------- | ---------- | ----------- |
+| 4.x.x | Beta | March 2021 | TBA | TBA |
+| 3.x.x | Active | February 2019 | TBA | TBA |
+| 2.x.x | Inactive | November 2015 | March 2020 | March 2021 |
```

Comment:
Should the deprecated and unsupported months for 2.x.x be Feb since 3.0.0 was released Feb 2019? But also since v2 is unsupported as of March this year, can we remove it from this table all-together?

Reply:
Yeh I went back and forth on this. I was hesitant to make the months for v2 February, since our deprecation policy technically wasn't released yet when v3 came out 🤔 .

But If v2 is definitely unsupported anyways, we can just update it to February. I don't see the harm in leaving it in the table, though. I know people are still using Android v2.

Reply:
Ah ok yeah that makes sense - I don't feel strongly either way about the dates then. We did remove public docs for it, so I'd say let's list it as unsupported in the table.

Reply:
Done!